app/controllers/callbacks_controller.rb

```diff
@@ -2,6 +2,6 @@ class CallbacksController < Devise::OmniauthCallbacksController
   def github
     @user = User.from_omniauth(request.env['omniauth.auth'])
     sign_in @user
-    redirect_to request.referer
+    redirect_to companies_path
```

Comment:
Uygulamaya iki yerden giriş yapabiliyorsun:

1. https://rubyturkiye.org/users
2. https://rubyturkiye.org/companies

Bu sebepten dolayı her seferinde `/companies`'e yönlendirme olmamalı. Benim aklıma gelen iki seçenek var:

1. Küçük bir geliştirme ile giriş yapılan sayfaya yönlendirip, kullanıcının kaldığı yerden devam etmesini sağlayabilirsin.
2. Hızlıca ilerlemeyi tercih edersen de `users/edit`'e yönlendirebilirsin.

Reply:
Aslında bunun için referrer kısmının session içinde saklanması yöntemi de kullanılıyor.